### PR TITLE
With great power comes great responsibility

### PR DESCRIPTION
**Pull request by Couls**
**What does this PR do:**
It adds a spiderman sprite and gives him a ranged web attack, credit to Benji64 for their art that can be found here: https://www.pixilart.com/art/spiderman-homemade-suit-5589401b2456da6
He can't be spawned by normal means.

**Images of sprite/map changes (IF APPLICABLE):**
image

ezgif-5-d1212f0da708

**Changelog:**
:cl:
add: spiderman
del: uncle ben
/:cl:

### CODE REVIEW

**Comment by Kyep** [comment]
Doesn't really fit the game. And the sprite looks terrible.

**Comment by Fox-McCloud** [comment]
...no

**Comment by Bookdrizzt** [comment]
Keep the web/webbing sprite and action because I remember an arachno-man gimmick that the admemes pulled once that would have been so cool with that, but god the spider-man sprite is horrendous.

**Comment by Couls** [comment]
Someone please make me a better sprite

**Comment by Couls** [comment]
With great spriting comes better chances?

**Comment by taukausanake** [comment]
I think it's cute, but I know no one is going to like it. Shame too

File: paradise.dme [hunk 1]
```diff
@@ -1899,6 +1900,7 @@
 #include "code\modules\mob\living\simple_animal\hostile\pirate.dm"
 #include "code\modules\mob\living\simple_animal\hostile\russian.dm"
 #include "code\modules\mob\living\simple_animal\hostile\spaceworms.dm"
+#include "code\modules\mob\living\simple_animal\hostile\spiderman.dm"
```
Comment (taukausanake) [inline paradise.dme]: Hmmmm? Is this supposed to be including itself in the .dme twice?

**Comment by Tayyyyyyy** [comment]
The other issue is that Benji64 doesn't list a license on that page, which means we can't assume they've given permission for their art to be used elsewhere. You should get explicit permission from the artist. It's not enough to just credit them unless it's released under a license like Creative Commons Attribution.

**Comment by AffectedArc07** [comment]
*sigh*

**Comment by taukausanake** [comment]
Wait, that little guy doesn't have directional sprites? :-1:

File: paradise.dme (hunk 1 above)
Comment (Couls) [reply paradise.dme]: Good catch

**Comment by Couls** [comment]
I'll probably do what @Bookdrizzt said and make it a function of the suit

**Comment by Pckables** [comment]
Well the sprite fits as well as the Sans sprite I've already seen in-game multiple times.

**Comment by Fox-McCloud** [comment]
>Well the sprite fits as well as the Sans sprite I've already seen in-game multiple times.

Blame admins for that. We wouldn't allow a Sans sprite in the code, itself.

**Comment by taukausanake** [comment]
I really want to see this Sans sprite now. I hate Undertale but this I want to see

**Comment by Pckables** [comment]
Surprise_crop

But lets not derail this.
The sprite is definitely going to be an issue for something thats going to have actual mechanics and function to.
I mean we already have a spiderman costume in-game.

**Comment by Couls** [comment]
I’ll figure something out for this later but until then I’ll close this PR

**Comment by Markolie** [comment]
A _Terror Spoder Man_, perhaps.

**Pull request closed without merging**